tools/pgroonga-primary-maintainer.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -xueo pipefail
+
+WAL_BLOCK_SIZE=8192
+
+reindex_threshold_size=0
+psql_command=$(which psql || :)
```

Comment:
`cd`しないから絶対パスじゃなくていいです。

```suggestion
psql_command=psql
```